src/replit_river/server_session.py

```diff
@@ -22,18 +21,18 @@
 
 from .rpc import (
     ACK_BIT,
-    STREAM_CLOSED_BIT,
     STREAM_OPEN_BIT,
     GenericRpcHandlerBuilder,
     TransportMessage,
     TransportMessageTracingSetter,
 )
 
-logger = logging.getLogger(__name__)
+STREAM_CLOSED_BIT = 0x0004  # Synonymous with the cancel bit in v2
```

Comment:
seems weird that this is defined away from where all the control flag bits are defined

Reply:
It's because it has a different meaning in v1 and v2